Supernovae_Ia/loadObservationalData.py

```diff
@@ -0,0 +1,114 @@
+"""
+Code from Matthieu Schaller to load observational SFR data.
```

Comment:
Should this be changed to something more realistic? I guess you wrote this!

Reply:
I changed this.